source/osmaxx/templates/base.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
In excerptexport app we used layouts (e.g. source/excerptexport/templates/excerptexport/layouts/base.html) for top level templates. This file looks like a top level template. Maybe you should move it to a layout directory.